src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+  for a WebSocket server or other plugins.
+- [angular-web-bluetooth] is a module for [Angular] that abstracts away all the
+  boilerplate needed to configure the Web Bluetooth API.
+- [&lt;platinum-bluetooth>] is a set of [Polymer] elements to discover and
+  communicate with nearby Bluetooth devices based on the Web Bluetooth API. For
+  instance, here's how to read battery level from a nearby bluetooth device
+  advertising a Battery service:
+  
+```js
+<platinum-bluetooth-device services-filter='["battery_service"]'>
+  <platinum-bluetooth-service service='battery_service'>
+    <platinum-bluetooth-characteristic characteristic='battery_level'>
+    </platinum-bluetooth-characteristic>
+  </platinum-bluetooth-service>
+</platinum-bluetooth-device>
+```
+
+```js
+const bluetoothDevice = document.querySelector('platinum-bluetooth-device');
+const batteryLevel = document.querySelector('platinum-bluetooth-characteristic');
+
+bluetoothDevice.request()
+.then(_ => batteryLevel.read())
+.then(value => {
+  console.log('Battery percentage is ' + value.getUint8(0));
+})
+.catch(error => { console.log(error); });
+```
+
+## Tools
+
+- [Get Started with Web Bluetooth] is a simple Web App that will generate all
+  the JavaScript boilerplate code to start interacting with a Bluetooth device.
+  Enter a device name, a service, a characteristic, define its properties and
+  you're good to go.
+- If you're already a Bluetooth developer, the [Web Bluetooth Developer Studio
+  Plugin] will also generate the Web Bluetooth JavaScript code for your
+  Bluetooth device.
+
+## Dev Tips
+
+A "Bluetooth Internals" page is available in Chrome at
+`chrome://bluetooth-internals` so that you can inspect everything about nearby
+Bluetooth devices: status, services, characteristics, and descriptors.
+
+<figure class="w-figure">
+  <img src="./bluetooth-internals.png" class="w-screenshot" alt="Screenshot of the internal page to debug Bluetooth in Chrome">
+  <figcaption class="w-figcaption">Internal page in Chrome for debugging Bluetooth devices.</figcaption>
+</figure>
+
+I would also recommend you check out the official ["How to file Web Bluetooth
+bugs"] page as debugging Bluetooth can be hard sometimes.
+
+## What's next
+
+Check the [browser and platform implementation status] first to know which parts
+of the Web Bluetooth API are currently being implemented.
+
+Though it's still incomplete, here's a sneak peek of what to expect in the near
+future:
+
+- [Scanning for nearby BLE advertisements](https://github.com/WebBluetoothCG/web-bluetooth/pull/239)
+  will happen with `navigator.bluetooth.requestLEScan()`.
+- A new `serviceadded` event will track newly discovered Bluetooth GATT Services
+  while `serviceremoved` event will track removed ones. A new `servicechanged`
+  event will fire when any characteristic and/or descriptor gets added or
+  removed from a Bluetooth GATT Service.
+
+{% YouTube '_BUwOBdLjzQ' %}
```

Comment:
Linking to a video from 2015 in a section called `Whats next?` sends a strange message. I'm OK with embedding the video, but perhaps you want to move it to a different section?

Reply:
I've moved it between resources and new acknowledgments section